vargrid.go

```diff
@@ -58,13 +60,16 @@ type VarGridConfig struct {
 	// PopConcThreshold is the limit for
 	// Σ(|ΔConcentration|)*combinedVolume*|ΔPopulation| / {Σ(|totalMass|)*totalPopulation}.
 	// See the documentation for PopConcMutator for more information.
-	PopConcThreshold float64
+	PopConcThreshold    float64
+	PopConcThresholdEnv string
```

Comment:
Yeah, I think I am against keeping this, but if we do we would definitely need more documentation.